tests/org.eclipse.swt.tests.win32/JUnit Tests/org/eclipse/swt/widgets/WidgetWin32Tests.java

```diff
@@ -0,0 +1,53 @@
+package org.eclipse.swt.widgets;
+
+import static org.junit.Assert.assertEquals;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.internal.DPITestUtil;
+import org.eclipse.swt.internal.DPIUtil;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class WidgetWin32Tests {
+	private Display display;
+	private boolean autoScaleOnRuntime;
+
+	@Before
+	public void setUp() {
+		autoScaleOnRuntime = DPIUtil.isAutoScaleOnRuntimeActive();
+		DPITestUtil.setAutoScaleOnRunTime(true);
+		display = Display.getDefault();
+	}
+
+	@After
+	public void tearDown() {
+		DPITestUtil.setAutoScaleOnRunTime(autoScaleOnRuntime);
+	}
+
+	@Test
+	public void widgetZoomShouldChangeOnZoomLevelChange() {
```

Comment:
Please name the test methods with the prefix `test`, it makes it easier to spot tests in the call hierarchy view.

Reply:
Done